src/js/apps/forms/form/form_app.js

```diff
@@ -180,6 +189,13 @@ export default App.extend({
       isEditing: !this.getState('responseId'),
     }));
   },
+  showLastUpdated(updated) {
+    if (this.isReadOnly || this.getState('responseId')) return;
```

Comment:
Last updated content will be hidden if the form is read only or has already been submitted.